README.md

```diff
@@ -1,47 +1,99 @@
 
-https://wiki.sugarlabs.org/
+For more information, visit:
+- [Sugar Labs Official Website](https://www.sugarlabs.org/)
+- [Sugar Labs Wiki](https://wiki.sugarlabs.org/)
 
-Installing on Debian or Ubuntu
-------------------------------
+---
 
-Automatically done when you install [Sugar
-desktop](https://github.com/sugarlabs/sugar).
+## Installation
 
-To install Sugar Toolkit alone without Sugar desktop,
+### Debian/Ubuntu
+Sugar Toolkit GTK3 is automatically installed with [Sugar desktop](https://github.com/sugarlabs/sugar).
+To install Sugar Toolkit alone without Sugar desktop:
 
-```
+```bash
 sudo apt install python3-sugar3
 ```
 
-Installing on Fedora
---------------------
-
-Automatically done when you install [Sugar
-desktop](https://github.com/sugarlabs/sugar).
+### Fedora
+Similar to Debian/Ubuntu, it is included with the [Sugar desktop](https://github.com/sugarlabs/sugar).
+To install Sugar Toolkit alone without Sugar desktop:
 
-To install Sugar Toolkit alone without Sugar desktop,
-
-```
+```bash
 sudo dnf install sugar-toolkit-gtk3
 ```
 
-Building
---------
-
-Sugar Toolkit follows the [GNU Coding
-Standards](https://www.gnu.org/prep/standards/).
-
-Install all dependencies, especially sugar-artwork and
-sugar-datastore.
+## Building from Source
+Sugar Toolkit GTK3 follows the [GNU Coding Standards](https://www.gnu.org/prep/standards/).
+
+### Note
+Ensure the following dependencies are installed:
+- `sugar-artwork`
+- `sugar-datastore`
+- `GTK+ 3`
+
+### Steps
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/sugarlabs/sugar-toolkit-gtk3.git
+   cd sugar-toolkit-gtk3
+   ```
+2. Run the configuration script:
+   ```bash
+   ./autogen.sh
+   ```
+3. Build and install:
+   ```bash
+   sudo make clean
+   sudo make
+   sudo make install
+   ```
+   
+Alternatively, you can skip the `make clean` step with:
+```bash
+sudo ./autogen.sh
+sudo make
+sudo make install
+```
 
-Clone the repository, run `autogen.sh`, then `make` and `make
-install`.
+---
+## Contributing
+
+We welcome contributions from the community! Follow these steps to get started:
+
+1. **Fork the Repository**:
+   ```bash
+   git clone https://github.com/sugarlabs/sugar-toolkit-gtk3.git
+   ```
+2. **Create a Feature Branch**:
+   ```bash
+   git checkout -b feature-name
+   ```
+3. **Make Your Changes** and **Commit**:
+   ```bash
+   git commit -m "Description of changes"
+   ```
+4. **Push Your Branch**:
+   ```bash
+   git push origin feature-name
+   ```
+5. **Submit a Pull Request**: Include a clear description of your changes and why they are beneficial.
+
+---
+Use the above process to setup and create activities, and contribute to the Sugar Labs community.
+
+## Community and Support
+
+If you have questions or need help, you can:
+- Join the [Sugar Labs Mailing List](https://lists.sugarlabs.org/)
+- Join the chat [Element Chat](https://app.element.io/#/room/#sugar:matrix.org)
```

Comment:
Element is a matrix client, not the only existing client so I'd rather you use Matrix that element.